F3FChrono/chrono/rs232_arduino.py

```diff
@@ -99,8 +99,10 @@ def receive(self):
                                 tmp = 0.
                                 for i in range(2, int(data[1])+2):
                                     tmp += int(data[i])/1000
-                                self.finaltime = tmp
+                                self.finaltime = int(data[12])/1000
```

Comment:
Maybe we could round it to 2 digits here ?
`Example
x = round(5.76543, 2)`

Reply:
In my point of view, I prefer store raw data with high precision in the database. 
I don't know...